rfc6962-bis.xml

```diff
@@ -903,21 +903,21 @@ messages.
         <t>
           POST https://&lt;log server&gt;/ct/v1/add-pre-chain
         </t>
-        <t>
-          Inputs:
-
-          <list>
-            <t><list style="hanging">
-              <t hangText="precertificate:">
-                The base64-encoded Precertificate.
-              </t>
-              <t hangText="chain:">
-                An array of base64-encoded CA certificates. The first element is
+	<t>
```

Comment:
Hmm. This is tabs vs. spaces. Possibly we should axe tabs in another CL, coz I'm sure there are many!